metis-indexing/src/main/java/eu/europeana/indexing/tiers/view/ContextualClasses.java

```diff
@@ -0,0 +1,46 @@
+package eu.europeana.indexing.tiers.view;
+
+import java.util.List;
+
+public class ContextualClasses {
+
+  private int completeContextualResources;
+  private int distinctClassesOfCompleteContextualResources;
+  private List<String> distinctClassesList;
+  private String metadataTier;
```

Comment:
Maybe instead of using a String we could use the values from an enum, like the ones being used in `SolrTier` (`RdfTier` and `EdmLabel`),  it would be one way to ensure there are no errors when writing the value as a String

Reply:
Not possible because:
- This is a view class and the tier class will be serialized as a whole, which is not what we want
- Also the "0" classification cannot be exactly represented in java because fields cannot be named as just a number.

It could be possible to do but I would need to merge the tiers somehow and write a custom (de)serializer I thnk.

Reply:
I think the closest we can get is `MediaTier` and `MetadataTier`, but still a (de)serializer would probably be required.
I'll give it some thought.

Reply:
I've managed to make something nice out of this.